sdcm/utils/gce_utils.py

```diff
@@ -30,11 +30,14 @@
 
 
 def append_zone(region: str) -> str:
-    assert region.startswith("us-east1"), "only `us-east1' region is supported"
-    if region.count("-") == 1:
+    assert region.startswith("us-east1") or region.startswith("us-west1"), "only supports two regions"
+    if region.count("-") != 1:
+        return region
+    if region.startswith("us-east1"):
         # us-east1 zones: b, c, and d. Details: https://cloud.google.com/compute/docs/regions-zones#locations
         return f"{region}-{random.choice('bccdd')}"  # choose zones c and d twice as often as zone b
-    return region
+    elif region.startswith("us-west1"):
+        return f"{region}-{random.choice('abc')}"
```

Comment:
Don't there are more than a, b and c?

Reply:
There are only 3 available zones for us-west1
- https://cloud.google.com/compute/docs/regions-zones